server/graphql_schemas/attend/schema.py

```diff
@@ -30,14 +33,20 @@ def mutate_and_get_payload(cls, root, info, **input):
         status = input.get('status')
         db_event_id = from_global_id(event_id)[1]
         event = Event.objects.get(id=db_event_id)
+        event_date = datetime.strptime(event.start_date+'00', '%Y-%m-%d %H:%M:%S.%f%z')
```

Comment:
Great job @dkam26. I think this line of code is error prone. The `strptime` function is unable to parse the date string we passed in as first argument. This results in a `ValueError` being thrown when the user tries to RSVP from the frontend. We can also see this from the snapshot tests for the attend schema mutations as they currently failing/not matching. I am guessing what we are trying to do here is to convert the event's start_date to a timezone-aware datetime (UTC) so that we can compare it with the value of `today`. To do that, I suggest using the timezone information of the event and then setup the datetime object as follows:
```
from pytz import timezone

...
event_tz = timezone(event.timezone)
event_start_date = datetime.strptime(event.start_date, '%Y-%m-%d %H:%M:%S.%f').replace(tzinfo=event_tz)
```